### PR TITLE
Added an entry for the accent-color CSS property

### PR DESCRIPTION
**Pull request by lukewarlow**
> What was wrong/why is this fix needed? (quick summary only)

This PR adds a page for the `accent-color` property from https://drafts.csswg.org/css-ui-4/#widget-accent.

PR to add the css formal definition: https://github.com/mdn/data/pull/490
PR to add the live example: https://github.com/mdn/interactive-examples/pull/1864
PR to add the browser compat data: https://github.com/mdn/browser-compat-data/pull/11470

This is my first time writing a full entry on MDN so apologies if I've missed a section that should be included.

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
@rachelandrew Could you take a look?
I can't see any glaring issues from the markup, but cannot assess the subject itself.

**Comment by teoli2003** [comment]
Should we add a glossary entry for _accent color_? (Or a little bit more information).

I had to google it to understand it (maybe because I'm not a native speaker).

**Comment by lukewarlow** [comment]
Perhaps including something like https://drafts.csswg.org/css-ui-4/#example-6760b48f would help make it clearer what it's referring to?

**Comment by teoli2003** [comment]
Yes, or a screenshot of one its use, though the live sample will do it, I think.

Maybe just a bit of text, maybe a paraphrase/simplificaation of this spec clause: "…a typically bright color that contrasts markedly with the more utilitarian background and foreground colors."

**Comment by lukewarlow** [comment]
I've made a glossary page with a rough description. Right now it does basically just say what the spec says. Not sure on how best to reword it to be more friendly.

File: files/en-us/web/css/accent-color/index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- Input
+- Reference
+- Styling HTML
+- accent-color
+- 'recipe:css-property'
+browser-compat: css.properties.accent-color
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>The <strong><code>accent-color</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property sets the color of the elements <a href="/en-US/docs/Glossary/accent">accent</a>. An accent appears in elements such as {{HTMLElement("input")}} of <code><a href="/en-US/docs/Web/HTML/Element/input/checkbox">type="checkbox"</a></code>, or <code><a href="/en-US/docs/Web/HTML/Element/input/radio">type="radio"</a></code>.</p>
```
Comment (teoli2003) [inline files/en-us/web/css/accent-color/index.html:18]: {{Glossary("accent")}}

**Comment by teoli2003** [comment]
Cool! I will let @rachelandrew do the formal review as she can land stuff in mdn/data at the same time (or before).

File: files/en-us/web/css/accent-color/index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- Styling HTML
+- accent-color
+- 'recipe:css-property'
+browser-compat: css.properties.accent-color
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>The <strong><code>accent-color</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property sets the color of the elements {{Glossary("accent")}}. An accent appears in elements such as {{HTMLElement("input")}} of <code><a href="/en-US/docs/Web/HTML/Element/input/checkbox">type="checkbox"</a></code>, or <code><a href="/en-US/docs/Web/HTML/Element/input/radio">type="radio"</a></code>.</p>
+
+<div>{{EmbedInteractiveExample("pages/css/accent-color.html")}}</div>
```
Comment (rachelandrew) [inline files/en-us/web/css/accent-color/index.html:20]: We wouldn't normally add an interactive example to something that requires flags to view it. So I suggest that you remove this until there are a couple of implementations.

**Review by rachelandrew — COMMENTED** [review]
This looks fine, however as it's an experimental property we won't show an interactive example right now as it won't work for most people, which is confusing.

File: files/en-us/web/css/accent-color/index.html (hunk 2 above)
Comment (lukewarlow) [reply files/en-us/web/css/accent-color/index.html]: Okay, I'll re-add in a follow up PR once it's shipped in Chrome.

File: files/en-us/glossary/accent/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: accent
+slug: Glossary/accent
+tags:
+  - Glossary
+  - Input
+  - accent
+---
+<p><span class="seoSummary">An <strong>accent</strong> is a typically bright color that contrasts with the more utilitarian background and foreground colors within a color scheme.</span> These are present in the visual style of many platforms (though not all).</p>
```
Comment (rachelandrew) [inline files/en-us/glossary/accent/index.html:9]:
```suggestion
<p>An <strong>accent</strong> is a typically bright color that contrasts with the more utilitarian background and foreground colors within a color scheme. These are present in the visual style of many platforms (though not all).</p>
```

**Review by rachelandrew — APPROVED** [review]
Thank you!
